### ISSUE TITLE
Evaluate tidyverse/broom

### ISSUE THREAD

**Issue by florianm** [carlopacioni/vortexR#36]
VortexR is https://github.com/tidyverse/broom for Vortex v10 output. 

If we can convert readLines to list of lists (similar to S3 classes), we could write a Vortex tidier for broom returning a "vortexr" class object.

Look at [broom tidiers](https://github.com/tidyverse/broom/blob/master/R/felm_tidiers.R) and see what we can learn.

**Comment by florianm** [comment]
probably more like [readLines to nested list](http://stackoverflow.com/a/6603126/2813717), then tidyverse through the list

**Comment by carlopacioni** [comment]
Good catch Florian, this can be used to improve Kendall test output (see #22). Possibly fit_regression (with the risk that memory allocation may become a problem with betareg). I still think that we should stream line data parsing with data.table as what we really want is a 'conventional' df that can be passed to ggplot and GLM functions

**Comment by florianm** [comment]
or a [tibble](https://blog.rstudio.org/2016/03/24/tibble-1-0-0/)!
